ack_timer.go

```diff
@@ -0,0 +1,109 @@
+package sctp
+
+import (
+	"sync"
+	"time"
+)
+
+const (
+	ackInterval float64 = 200 // msec
+)
+
+// ackTimerObserver is the inteface to an ack timer observer.
+type ackTimerObserver interface {
+	onAckTimeout()
+}
+
+// ackTimer provides the retnransmission timer conforms with RFC 4960 Sec 6.3.1
+type ackTimer struct {
+	observer ackTimerObserver
+	interval float64
```

Comment:
maybe this can be a time.Duration instead?

Reply:
Agreed.

Reply:
Done